src/components/Frame/Frame.scss

```diff
@@ -218,6 +218,7 @@ $skip-vertical-offset: rem(10px);
 }
 
 .Content {
+  position: relative;
```

Comment:
I agree with @kyledurand - this change is innocent enough and shouldn't cause problems. I have not tophatted, but I trust that this is a viable solution.